aries_cloudagent/protocols/coordinate_mediation/v1_0/manager.py

```diff
@@ -99,6 +126,17 @@ async def grant_request(self, mediation: MediationRecord) -> MediationGrant:
         )
         return grant
 
+    async def granted_request(self, mediation: MediationRecord,
+                              endpoint,
+                              routing_did_verkey: Sequence[str]
+                              ) -> MediationRecord:
+        """update Mediation state to granted."""
+        # TODO: ?create if not existing?
+        mediation.state = MediationRecord.STATE_GRANTED
+        await mediation.save(self.context, reason="Mediation request granted")
+        # TODO: update endpoint/keylists from parameters
+        return mediation
+
```

Comment:
For now, let's leave this as a stub.

Reply:
I need this for mediation_granted_handler.py where a client aca-py has received a grant message and needs to update his state. 
Is there something you don't like I can address with the method? I could move the logic directly into the handler but that will break the manager pattern we have.